include/couch_replicator_api_wrap.hrl

```diff
@@ -22,10 +21,9 @@
     timeout,            % milliseconds
     ibrowse_options = [],
     retries = 10,
-    wait = 250,         % milliseconds
+    wait = [250, 25],  % [normal error wait time, 429 backoff wait time]
```

Comment:
Ya I considered it as well. I think I was a little angry with all the tuple matching issues this past  week and got annoyed with tuples lol. I am ok with changing it to a tuple.